### PR TITLE
Escape

### PR DESCRIPTION
**Pull request by jacknoble**
Addresses https://github.com/rorysaur/quack/issues/47

### CODE REVIEW

File: js/components/input.jsx [hunk 1]
```diff
@@ -33,7 +33,6 @@ var Input = React.createClass({
   _onChange: function(event) {
     this.setState({
       message: event.target.value,
-      commandCharacter: this.state.commandCharacter
```
Comment (jacknoble) [inline js/components/input.jsx:36]: setState merges the state object so this isn't necessary.

File: js/utils/user_command_handler.js [hunk 2]
```diff
@@ -32,7 +32,7 @@ module.exports = {
         actions.renameLocalUser(args[0]);
       },
 
-      config: function() {
+      set: function() {
```
Comment (jacknoble) [inline js/utils/user_command_handler.js:35]: this seems better

File: js/stores/settings_store.js [hunk 3]
```diff
@@ -4,7 +4,8 @@ var AppDispatcher = require('../dispatcher/app_dispatcher');
 var ActionTypes = require('../constants/constants').ActionTypes;
 
 var _settings = {
-  cmd: ":"
+  cmd: ':',
+  escape: '\\'
```
Comment (rorysaur) [inline js/stores/settings_store.js:8]: escape is 2 backslashes but you only slice 1?

Comment (jacknoble) [inline js/stores/settings_store.js:8]: Escape is one backslash but you have to escape it in the code or else it escapes the closing quote.

Comment (rorysaur) [inline js/stores/settings_store.js:8]: oh haha.. i forgot.

**Pull request merged**